models/games.models.js

```diff
@@ -67,3 +67,24 @@ exports.selectCommentsByReviewId = (review_id) => {
       });
   });
 };
+
+exports.insertComment = (review_id, body, author) => {
+  return checkExists("users", "username", author).then(() => {
+    return checkExists("reviews", "review_id", review_id).then(() => {
```

Comment:
You shouldn't be nesting .thens - they should be chained onto the end of the last one.